thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftCallService.java

```diff
@@ -80,6 +80,10 @@ public static ThriftCallService of(Object implementation) {
      *                        {@code *.Iface} or {@code *.AsyncIface} service interface generated by
      *                        the Apache Thrift compiler
      */
+    public static ThriftCallService ofMultiple(Map<String, ? extends Iterable<?>> implementations) {
```

Comment:
There is erasure compilation error cause. So had to change the method name. Let me know if this is fine, or any better way to do this that I'm missing.